portfolio/src/main/webapp/js/script.js

```diff
@@ -41,10 +41,10 @@ function switchIcon(icon) {
 }
 
 /**
- * Display text from data Servlet
+ * get Number of comments from data servlet
  */
-async function getText() {
-  const response = await fetch('/data');
+async function getNumComments(numComments) {
+  const response = await fetch('/data?numComments='.concat(numComments));
```

Comment:
try this instead of concat
```
`/data?numComments=${numComments}`
```

Reply:
I'm not sure why, but I tried this before and for some reason it doesn't work. Do you know why that could be?

Reply:
Make sure you are using a backtick, not a single quote. Only the backtick version has special string substitution semantics.

Reply:
That was the problem, thanks!